chess/engine.py

```diff
@@ -1246,6 +1246,26 @@ async def analysis(self, board: chess.Board, limit: Optional[Limit] = None, *, m
         and stopping the analysis at any time.
         """
 
+    @abc.abstractmethod
+    async def send_game_result(self, board: chess.Board, winner: Optional[Color] = None, game_ending: Optional[str] = None, game_complete: bool = True) -> None:
+        """
+        Sends the engine the result of the game.
+
+        UCI engines only recieve the final position of the board. XBoard engines
```

Comment:
Mhh ... this sounds like this method would send it, but if I understand correctly it doesn't.